services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HankeFactory.kt

```diff
@@ -55,6 +57,12 @@ class HankeFactory(
     }
 
     fun save(hanke: Hanke) = hankeService.createHanke(hanke)
+    fun saveMinimal(hankeTunnus: String): HankeEntity =
```

Comment:
Nitpicking, but a line break would be nice.

Reply:
Furthermore, could this be:

    fun saveMinimal(hankeTunnus: String = hanketunnusService.newHanketunnus()): HankeEntity =
        hankeRepository.save(HankeEntity(hankeTunnus = hankeTunnus))

No need for overloading then.